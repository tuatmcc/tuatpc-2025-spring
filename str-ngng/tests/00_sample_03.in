20 2
abababababababababab
1 20
1 16
