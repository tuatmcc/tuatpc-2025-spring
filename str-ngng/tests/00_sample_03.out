24
36
9
12
6
